R/data_validation.R

```diff
@@ -64,14 +66,16 @@ confirm_sufficient_data <- function(epi_data, ahead, args_input, buffer = 9) {
   # TODO: Buffer should probably be 2 * n(lags) * n(predictors). But honestly,
   # this needs to be fixed in epipredict itself, see
   # https://github.com/cmu-delphi/epipredict/issues/106.
-
+  if (extra_sources == c("")) {
+    extra_sources <- character(0L)
+  }
+  has_no_last_nas <- epi_data %>%
+    drop_na(c(!!outcome, !!!extra_sources)) %>%
```

Comment:
How does this line help in practice? Do you just only specify "hhs" here and ignore "chng"?

Reply:
until `chng` gets put into `extra_sources` it will get ignored. An example of this working as intended can be found in this test: 
https://github.com/cmu-delphi/exploration-tooling/blob/9df36ba981270e7bb778a5d2af71151b31fc1d9b/tests/testthat/test-forecasters-basics.R#L75-L89